rslib/src/decks/remove.rs

```diff
@@ -37,7 +37,7 @@ impl Collection {
             // if the default deck is included, just ensure it's reset to the default
             // name, as we've already removed its cards
             let mut modified_default = deck.clone();
-            modified_default.name = self.tr.deck_config_default_name().into();
+            modified_default.name = NativeDeckName(self.tr.deck_config_default_name().into());
```

Comment:
Can we assume fluent strings are normalised?

Reply:
Would be safest not to assume they're normalized